packages/web-components/src/tabs/tabs.styles.ts

```diff
@@ -0,0 +1,258 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  colorNeutralForeground1,
+  colorNeutralForeground1Hover,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundPressed,
+  curveDecelerateMax,
+  durationSlow,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalMNudge,
+  spacingHorizontalSNudge,
+  spacingVerticalL,
+  spacingVerticalMNudge,
+  spacingVerticalS,
+  spacingVerticalSNudge,
+  spacingVerticalXXS,
+  strokeWidthThicker,
+} from '../theme/design-tokens.js';
+import { TabTokenNames } from './tabs.js';
+
+export const styles = css`
+  :host {
+    display: grid;
+    box-sizing: border-box;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    color: ${colorNeutralForeground2};
+    grid-template-columns: auto 1fr auto;
+    grid-template-rows: auto 1fr;
+  }
+
+  :host([disabled]) {
+    cursor: not-allowed;
+    color: ${colorNeutralForegroundDisabled} !important;
+  }
+
+  :host([disabled]) ::slotted(fluent-tab) {
```

Comment:
The entire tabs component can be disabled?

Reply:
Yes, oddly, yes